ceramic-kubo-rpc/src/error.rs

```diff
@@ -0,0 +1,21 @@
+//! Provides simple error type for communicating specific failures.
+use thiserror::Error;
+
+/// Error type for specific failures.
+/// These generally come from here
+/// https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+/// However we start with a minimal set and can add more as needed.
+#[derive(Debug, Error)]
+pub enum Error {
```

Comment:
Do we need these specific error types? I think we can do something like https://www.reddit.com/r/rust/comments/ozc0m8/an_actixanyhow_compatible_error_helper_i_found/?utm_source=share&utm_medium=web2x&context=3 if we need to convert to actix easily.

Reply:
We don't _need_ these yet but as a general design I prefer libraries to use an error enum instead of anyhow so that consumers of the lib can handle specific errors as needed in their application. So while this error enum is very light right now the intent is to let it grow as the needs of the lib grow.